common-healing.lic

```diff
@@ -249,13 +249,34 @@ module DRCH
   end
 
   def bind_wound(body_part, person = 'my')
+    # Either you tended it or it already has bandages on. Good job!
+    tend_success = [
+      'You work carefully at tending',
+      'That area has already been tended to',
+      'That area is not bleeding'
+    ]
+    # Yeouch, you couldn't tend the wound. Might have made it worse!
+    tend_failure = [
+      'You fumble',
+      'too injured for you to do that'
+    ]
+    # You dislodged ammo or a parasite, discard it.
+    tend_dislodge = [
+      'You \w+ remove'
+    ]
     snap = [DRC.left_hand, DRC.right_hand]
-    if /You \w+ remove/ =~ DRC.bput("tend #{person} #{body_part}", 'You work carefully at tending', 'That area has already been tended to', 'That area is not bleeding', 'You fumble', 'too injured for you to do that', 'You \w+ remove')
+    result = DRC.bput("tend #{person} #{body_part}", *tend_success, *tend_failure, *tend_dislodge)
+    waitrt?
+    case result
+    when *tend_dislodge
       DRCI.dispose(DRC.left_hand) if DRC.left_hand != snap.first
       DRCI.dispose(DRC.right_hand) if DRC.right_hand != snap.last
       bind_wound(body_part, person)
+    when *tend_failure
+      false
+    else
+      true
```

Comment:
The goal of this refactoring -- to know if _fail_ to tend the bleeder. `healme` script wants to track successfully tended bleeders, or bleeders that don't need to be tended again until the bandages fall off. So if we fail to tend then `healme` won't ignore that bleeder next attempt.